server/datastore/mysql/mysql.go

```diff
@@ -62,6 +62,22 @@ type Datastore struct {
 
 type txFn func(sqlx.ExtContext) error
 
+type entity struct {
+	name string
+}
```

Comment:
Thanks for the note on why to make this a struct! I hadn't considered that Go would automatically type a string literal as this type if we used my proposed approach.